.eslintrc.json

```
@@ -33,8 +33,24 @@
     "jest",
     "import",
     "eslint-plugin-tsdoc",
-    "prettier"
+    "prettier",
+    "markdown"
   ],
+
+  "overrides": [
```

Biome (1.9.4)

[error] The key overrides was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)